parsons/action_network/action_network.py

```diff
@@ -43,58 +46,33 @@ def _get_entry_list(self, object_name, limit=10, timeout=60):
             if limit:
                 if count >= limit:
                     return Table(return_list[0:limit])
-            if time() - t0 > timeout:
-                logger.info("Request timed out. Returning results so far.")
-                return Table(return_list)
-
-    def _get_entry(self, object_name, object_id):
-        # returns a specific entry by object name and id
-        return self.api.get_request(url=f"{self.api_url}/{object_name}/{object_id}")
-
-    def _add_entry(self, object_name, data, silent=False):
-        # adds an entry to action network
-        response = self.api.post_request(url=f"{self.api_url}/{object_name}", data=json.dumps(data))
-        if not silent:
-            identifiers = response['identifiers']
-            entry_id = [entry_id.split(':')[1]
-                        for entry_id in identifiers if 'action_network:' in entry_id][0]
-            logger.info(f"Entry {entry_id} successfully added to {object_name}")
-
-    def _update_entry(self, object_name, entry_id, data, silent=False):
-        # updates fields for a given entry
-        # only the fields to be updated need to be included
-        response = self.api.put_request(url=f"{self.api_url}/{object_name}/{entry_id}",
-                                        json=json.dumps(data), success_codes=[204, 201, 200])
-        if not silent:
-            logger.info(f"{object_name.capitalize()} entry {entry_id} successfully updated")
-        return response
 
-    def get_people_list(self, limit=10, timeout=60):
+    def get_people(self, limit=None, per_page=25, page=None):
```

Comment:
Could you add documentation in the `Args` below for the two new arguments? Same for `get_tags`.

Reply:
Wow, that was careless on my part. Thanks for catching that.